pkg/restore/db.go

```diff
@@ -163,9 +164,11 @@ func (db *DB) CreateTable(ctx context.Context, table *utils.Table) error {
 			utils.EncloseName(table.Db.Name.O),
 			utils.EncloseName(table.Info.Name.O),
 			table.Info.AutoIncID)
+		if table.NeedRebaseAutoID() {
```

Comment:
🤔  why not just use `NeedAutoID(table.Info)` here?

Reply:
Because this function isn't written in the first version 🧐.

You'r right...